_episodes/13-live.md

```diff
@@ -250,6 +250,10 @@ you are unsure of the effect you behaviour has on the workshop.
 > have found that it's difficult to capture both the person and the
 > screen with a handheld device---but give feedback the same way you
 > have previously (positive and negative, content and presentation).
+> If you decide to teach something else, make sure to choose something
+> you know well, and prepare the material in advance.
+> Also, explain in advance to your fellow trainee what you will be teaching
+> and what the learners you teach it to are expected to be familiar with.
```

Comment:
We are going to ask people to read one lesson episode carefully in advance of the training (see #246) which will necessitate changes here.